js/_website/generate_jsons/src/docs/__init__.py

```diff
@@ -54,35 +55,26 @@ def add_demos():
 
 add_demos()
 
-ordered_events = [
-    "Change()",
-    "Click()",
-    "Submit()",
-    "Edit()",
-    "Clear()",
-    "Play()",
-    "Pause()",
-    "Stream()",
-    "Blur()",
-    "Upload()",
-]
-
-
-def add_supported_events():
+def create_events_matrix():
+    events = []
+    for c in EventListener.__subclasses__():
+        methods = c().__dict__
+        for m in methods: 
+            if m[:1] != '_' and isinstance(methods[m], EventListenerMethod) and m[:1] not in events: 
```

Comment:
I think you mean `m not in events` here?

Reply:
Otherwise looks good!

Reply:
lol